pkgs/development/tools/build-managers/bazel/bazel-deps/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv, buildBazelPackage, lib, fetchFromGitHub, git, jre, makeWrapper }:
+
+buildBazelPackage rec {
+  name = "bazel-deps";
+  version = "2018-05-31";
```

Comment:
When we use a date as the version we don't put the hyphens in it, because hyphens have other meanings in the names. Can you use 20180531?

Reply:
No, “The date must be in "YYYY-MM-DD" format.”: https://nixos.org/nixpkgs/manual/#sec-package-naming. (There is also an "-unstable" suffix requirement, but it seems better to skip it for packages that do not plan to do versioned releases.)